### PR TITLE
Use standard logger in post_processor, return full path as fallback i…

### PR DESCRIPTION
**Pull request by medariox**
- [x] PR is based on the DEVELOP branch
- [x] Don't send big changes all at once. Split up big PRs into multiple smaller PRs that are easier to manage and review
- [x] Read the [contribution guide](https://github.com/pymedusa/Medusa/blob/master/.github/CONTRIBUTING.md)

…n get_rel_path, lint

### CODE REVIEW

File: medusa/post_processor.py [hunk 1]
```diff
@@ -129,32 +122,28 @@ def _checkForExistingFile(self, existing_file):
...
+            logger.debug(u"There is no existing file so there's no worries about replacing it")
             return PostProcessor.DOESNT_EXIST
 
         # if the new file exists, return the appropriate code depending on the size
         if os.path.isfile(existing_file):
-
             # see if it's bigger than our old file
             if os.path.getsize(existing_file) > os.path.getsize(self.file_path):
-                self._log(u'File {0} is larger than {1}'.format(existing_file, self.file_path), logger.DEBUG)
+                logger.debug(u'File {0} is larger than {1}'.format(existing_file, self.file_path))
```
Comment (ratoaq2) [inline medusa/post_processor.py]: You can use lazy logging: without formatting the msg:

`logger.debug('My msg {abc}',  abc='def')`

File: medusa/post_processor.py [hunk 2]
```diff
@@ -83,22 +87,10 @@ def __init__(self, file_path, nzb_name=None, process_method=None, is_priority=No
...
-    def _log(self, message, level=logger.INFO):
-        """
-        A wrapper for the internal logger which also keeps track of messages and saves them to a string for later.
-
-        :param message: The string to log (unicode)
-        :param level: The log level to use (optional)
-        """
-        logger.log(message, level)
-        self.log += message + '\n'
-
```
Comment (fernandog) [inline medusa/post_processor.py]: @medariox @ratoaq2 without this we can't show the result logs in manual PP page. 

you know it right?

File: medusa/post_processor.py (hunk 1 above)
Comment (medariox) [inline medusa/post_processor.py]: I know, my eyes were bleeding.

File: medusa/post_processor.py (hunk 2 above)
Comment (medariox) [inline medusa/post_processor.py]: It just shows the important stuff now and I would even reduce it further. The rest can be found in the logs.

Comment (fernandog) [inline medusa/post_processor.py]: Did you tried with nzbtomedia ? cause it also shows medusa log there from self.log

Comment (medariox) [inline medusa/post_processor.py]: I just removed the ones in post_processor, doesn't affect the other self._log. Thinking about it we might want to keep some more messages

**Comment by medariox** [comment]
No time to finish this.

**Pull request closed without merging**